src/plugins/gui/scenes/SepaFormScene.tsx

```diff
@@ -64,40 +62,28 @@ export const SepaFormScene = React.memo((props: Props) => {
 
   return (
     <SceneWrapper hasNotifications>
```

Comment:
Can it be `<SceneWrapper scroll>` so we don't lose the scroll feature? It won't be keyboard aware, of course.